packages/nimble-components/src/spinner/tests/spinner.stories.ts

```diff
@@ -22,11 +23,28 @@ const spinnerSize = {
 
 interface SpinnerArgs {
     size: keyof typeof spinnerSize;
+    appearance: keyof typeof SpinnerAppearance;
 }
 
 const overviewText = '<p>The `nimble-spinner` is an animating indicator that can be placed in a particular region of a page to represent '
```

Comment:
Instead of putting this docs content in the `.stories.ts` file, let's keep the story limited to just the component demo and API docs. Instead you can add a `.mdx` file for the docs. That's a markdown format so it'll be easier to maintain text content than in HTML. And it allows embedding the interactive demo and API docs anywhere within the file. (This is a new pattern we've started adopting recently but eventually I hope all of our components will use this approach).

You can see a great example of this in [Malcolm's label-provider PR](https://github.com/ni/nimble/pull/1328/files#diff-b240e51b1ed36cc5b8c6f4bcd4fd6b1d22dd5006bc1c86618a132215cc2b2c3c) and some simpler examples in other .mdx files already in the repo.

Once you do this I'd suggest a document structure something like:
1. intro paragraph (maybe lines 29-31 of your current docs?)
2. interactive demo and API table
3. Sections for sizing / appearance / accessibility